axum-extra/src/extract/multipart.rs

```diff
@@ -43,7 +43,7 @@ use std::{
 ///         let name = field.name().unwrap().to_string();
 ///         let data = field.bytes().await.unwrap();
 ///
-///         println!("Length of `{}` is {} bytes", name, data.len());
+///         println!("Length of `{name}` is {} bytes", data.len());
```

Comment:
For cases like this where there are some inline args and some not I prefer to no inline any of them.

Reply:
fixed